olive/systems/azureml/aml_system.py

```diff
@@ -373,6 +385,38 @@ def pass_runner_pipeline():
 
         return pass_runner_pipeline()
 
+    def _get_data_configs(self, pass_config: dict):
```

Comment:
Instead of hard coding `train_data_config` and `eval_data_config`, we should generalize the support for data configs like we do for path and data params. There are other passes that also have data configs. 

We could do something similar to `path_params` https://github.com/microsoft/Olive/blob/main/olive/passes/olive_pass.py#L92

Reply:
Besides train data config and eval data config, which kind of data config may be used in the passes?

Reply:
https://github.com/search?q=repo%3Amicrosoft%2FOlive+path%3A%2F%5Eolive%5C%2Fpasses%5C%2F%2F+DataConfig&type=code

Reply:
Will update this.